carla_ai/WaypointWithSpeedLimit[] path
